Misc/NEWS.d/next/Library/2018-10-25-09-59-00.bpo-35047.abbaa.rst

```diff
@@ -0,0 +1,3 @@
+``unittest.mock`` now includes mock calls in exception messages if
```

Comment:
Nitpick: the change affects also `assert_called_once_with` and `assert_has_calls`

Reply:
`assert_has_calls` was only refactored. Is it OK to write `assert_called_once(_with)`?

Reply:
Good point!

About assert_called_once_with, I'd just list the three

Reply:
Done!